Content.Shared/_DV/Weapons/Ranged/Systems/SharedGunSystem.Bipod.cs

```diff
@@ -0,0 +1,219 @@
+using System.Linq;
+using Content.Shared._DV.Weapons.Ranged.Components;
+using Content.Shared.Actions;
+using Content.Shared.DoAfter;
+using Content.Shared.Hands;
+using Content.Shared.Hands.Components;
+using Content.Shared.IdentityManagement;
+using Content.Shared.Interaction.Events;
+using Content.Shared.Maps;
+using Content.Shared.Toggleable;
+using Content.Shared.Weapons.Ranged.Events;
+using Robust.Shared.Physics.Systems;
+using Robust.Shared.Serialization;
```

Comment:
unused usings `Content.Shared.Maps` and `Robust.Shared.Physics.Systems`

```suggestion
using Content.Shared.Hands.Components;
using Content.Shared.IdentityManagement;
using Content.Shared.Interaction.Events;
using Content.Shared.Toggleable;
using Content.Shared.Weapons.Ranged.Events;
using Robust.Shared.Serialization;
```

Reply:
You are a hawk. Removed them! 👍